sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,560 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Amazon SageMaker Feature Store: Encrypt Data in your Online or Offline Feature Store using KMS key"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates how to enable encyption for your data in your online or offline Feature Store using KMS key. We start by showing how to programmatically create a KMS key, and how to apply it to the feature store creation process for data encryption. The last portion of this notebook demonstrates how to verify that your KMS key is being used to encerypt your data in your feature store.\n",
```

Comment:
"This notebook demonstrates how to enable encryption for your data in your online or offline feature store using a KMS key. We start by showing how to programmatically create a KMS key and how to apply it to the feature store creation process for data encryption. The last portion of this notebook demonstrates how to verify that your KMS key is being used to encrypt your data in your feature store.\n",